examples/python/performance_test.py

```diff
@@ -1,22 +1,28 @@
 import asyncio
 import json
 from sys import argv
+import time
+
 import websockets
-import traceback
-from authenticate import authenticate_user
+
+from authentication import create_get_id_token
 from cusips_to_figis import openfigi_map_cusips_to_figis
 
 async def main():
-    if len(argv) != 4:
-        print('Usage: python api_client_timestamp_sample.py <Deep MM dev username> <password> <openfigi_api_key>')
+    if len(argv) != 6:
```

Comment:
I think this is fine -- we just need to put in the documentation which region and ID token to use (for dev since we're still on staging1 for the API until I build the new servers).

Reply:
Maybe in a comment here we should point them to the README which has those values they can use.